lookout/style/format/virtual_node.py

```diff
@@ -116,15 +116,16 @@ def from_node(node: bblfsh.Node, file: str, path: str,
         :return: New VirtualNode-s.
         """
         outer_token = file[node.start_position.offset:node.end_position.offset]
+        if outer_token == "''" or outer_token == '""':
```

Comment:
🤔 why?

Reply:
Because that how it works for the new driver. You have a token there and for v1 driver you didn't. 
From the assent below you can see that I expect empty token or `""` or `''`.
Empty was for v1.2.0 driver and I assume can happen again for any new language we add to style-analyzer so I keep it.

`""` and `''` are expected from JS driver v2.7.1